backend/entities/studyPlan.entity.ts

```diff
@@ -26,14 +31,27 @@ export class StudyPlan {
   @UpdateDateColumn({ select: false })
   updatedAt!: Date;
 
-  @OneToOne(() => User, (user) => user.studyPlan)
-  user!: Relation<User>;
-
   @OneToMany(() => Semester, (semester) => semester.studyPlan, {
     cascade: ["remove"],
   })
   semesters!: Relation<Semester>[];
 
+  @OneToMany(
+    () => StudyPlanCollaborator,
+    (studyPlanCollaborator) => studyPlanCollaborator.studyPlan,
+    {
+      cascade: ["remove"],
+    },
+  )
+  studyPlanCollaborator!: Relation<StudyPlanCollaborator>[];
```

Comment:
why is this singular `studyPlanCollaborator` instead of plural `studyPlanCollaborators`